qutip/core/superop_reps.py

```diff
@@ -59,7 +59,13 @@
 from .states import basis
 
 
-_SINGLE_QUBIT_PAULI_BASIS = (identity(2), sigmax(), sigmay(), sigmaz())
+# TODO: revisit when creation routines have dispatching.
+_SINGLE_QUBIT_PAULI_BASIS = (
+    identity(2).to(_data.CSR),
```

Comment:
Wouldn't it be better to have these as `Dense` per default?

Reply:
Possibly?  They're only used in the next function down in `superop_reps`, and I didn't test enough of what it was doing to know if a sparse or dense representation was better for that function.  It looks sort of sparse, maybe?